### ISSUE TITLE
Batch Norm operation needs transpose of input shape

### ISSUE THREAD

**Issue by pramod-wick** [xieyulai/LSTM-for-Human-Activity-Recognition-using-2D-Pose_Pytorch#2]
Referring to the post given here (https://discuss.pytorch.org/t/batchnorm1d-input-shape/91364). Batch norm expects a shape of  [batch_size, features, timesteps], however the format used in lstm.ipynb is [batch_size, timesteps, features]. Thereby a transpose operation is required as shown in the post above.
